README.md

```diff
@@ -57,9 +57,9 @@ node versus a single A100 GPU (using WarpDrive), for the Tag environment with 10
 
 
 ## Code Structure
-WarpDrive provides a CUDA + Python framework and quality-of-life tools so you can quickly build fast and flexible multi-agent RL systems. The following figure illustrates a bottoms-up overview of the design and components of WarpDrive. The user only needs to write a CUDA step function at the CUDA environment layer, while the rest is a pure Python interface. We have step by step tutorials for you to master the workflow.
+WarpDrive provides a CUDA + Python framework and quality-of-life tools so you can quickly build fast, flexible and massively distributed multi-agent RL systems. The following figure illustrates a bottoms-up overview of the design and components of WarpDrive. The user only needs to write a CUDA step function at the CUDA environment layer, while the rest is a pure Python interface. We have step by step tutorials for you to master the workflow.
 
-<img src="https://user-images.githubusercontent.com/31748898/132800991-e181bba6-b559-4b54-b7fe-461424765ebc.png" width="780" height="510"/>
+<img src="https://user-images.githubusercontent.com/31748898/150910647-f939a9b2-0b3f-41f1-a9da-6418a6485f1b.png" width="780" height="580"/>
```

Comment:
Looks good. I only have a couple of suggestions - change `distributed layer` to `distributed training layer`. Also, the paths to the auto scaler and distributed trainer are not quite right.